src/MusicStore/Models/ShoppingCart.cs

```diff
@@ -93,9 +93,11 @@ public void EmptyCart()
         public async Task<int> GetCount()
         {
             // Get the count of each item in the cart and sum them up
-            return await (from cartItem in _dbContext.CartItems
-                          where cartItem.CartId == ShoppingCartId
-                          select cartItem.Count).SumAsync();
+            return await _dbContext
+                .CartItems
+                .Where(c => c.CartId == _shoppingCartId)
+                .Select(c => c.Count)
+                .SumAsync();
         }
 
         public async Task<decimal> GetTotal()
```

Comment:
Same. In general methods for which the implementation returns a Task at the tail don't need to be async, just return a Task. And this is showcasing a good practice :smile: